backend/apps/beaver_api/beaver_api/beaver_api/urls.py

```diff
@@ -27,6 +34,26 @@
         include("rest_framework.urls", namespace="rest_framework"),
     ),
     path("api/v1/", include("code_api.urls"), name="code_api"),
+    path(
+        "api/v1/bulk_create/",
+        CodeDocumentCreateView.as_view(),
+        name="code-document-bulk-create",
+    ),
+    path(
+        "api/v1/bulk_update/",
+        CodeDocumentUpdateView.as_view(),
+        name="code-document-bulk-update",
+    ),
+    path(
+        "api/v1/bulk_delete/",
+        CodeDocumentDeleteView.as_view(),
+        name="code-document-bulk-delete",
+    ),
+    path(
+        "api/v1/code_documents/",
+        CodeDocumentListView.as_view(),
+        name="code-documents-list",
+    ),
```

Comment:
that should be declared to the code_api
please see the line above ' path("api/v1/", include("code_api.urls"), name="code_api"),'